chart/templates/webhook.yaml

```diff
@@ -107,6 +107,37 @@ webhooks:
     resources:
     - disruptions
 ---
+apiVersion: admissionregistration.k8s.io/v1beta1
+kind: MutatingWebhookConfiguration
```

Comment:
What is the purpose of this webhook btw?
We are trying to limit the number of mutating webhooks internally as they could impact the entire cluster.

Reply:
Ah, the project uses v1beta1 for CRDs which doesn't support default values.

Reply:
Indeed, that's why. We'll be moving off a mutating webhook once we're on v1, which should be by eoy